<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright (c) Kitware Inc. -->
<!-- SPDX-License-Identifier: BSD-3-Clause -->
<ui version="4.0">
 <class>VRCollaborationWidget</class>
 <widget class="QWidget" name="VRCollaborationWidget">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="cHeader">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Collaboration</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cCollabEnable">
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="configureAvatarBtn">
     <property name="toolTip">
      <string>Map events to avatar head/hands.</string>
     </property>
     <property name="text">
      <string>Edit Avatar</string>
     </property>
     <property name="icon">
      <iconset resource="../../Qt/Components/Resources/pqComponents.qrc">
       <normaloff>:/pqWidgets/Icons/pqWrench.svg</normaloff>:/pqWidgets/Icons/pqWrench.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cServerLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The IP address or server name to connect to</string>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="cServerValue">
     <property name="text">
      <string>vrserver.kitware.com</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="cSessionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>What session to connect to. A session is a string name. People who join the same session on the same server will see each other.</string>
     </property>
     <property name="text">
      <string>Session</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="cSessionValue">
     <property name="text">
      <string>pv</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="cNameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Your name to appear in XR</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="cNameValue">
     <property name="text">
      <string>Anonymous</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="cPortLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>What port to connect to</string>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="cPortValue">
     <property name="text">
      <string>5555</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="cAvUpLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Default up vector for new avatars</string>
     </property>
     <property name="text">
      <string>Default Avatar Up</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="cAvatarUpX">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="cAvatarUpY">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="cAvatarUpZ">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="outputWindow"/>
   </item>
  </layout>
 </widget>
 <resources/>
</ui>
